api/types/types.go

```diff
@@ -167,11 +186,11 @@ type Info struct {
 	DriverStatus       [][2]string
 	MemoryLimit        bool
 	SwapLimit          bool
-	CpuCfsPeriod       bool
-	CpuCfsQuota        bool
+	CPUCfsPeriod       bool
```

Comment:
Same as above, as it is *serialized* as JSON, I think it would be better to do something like :

```go
CPUCfsPeriod bool `json:CpuCfsPeriod`
CPUCfsQuota bool `json:CpuCfsQuota`
```